xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/ActionFilter.java

```diff
@@ -84,16 +86,25 @@ public void init(FilterConfig filterConfig) throws ServletException
     @SuppressWarnings("unchecked")
     @Override
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException,
-        ServletException
+            ServletException
```

Comment:
Was this change done with automatic formatting? It doesn't look right. Did you configure you IDE to use the [XWiki code style](https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/)?

Reply:
I have indeed used the automatic indentation. I'm using IntelliJ with the codestyle-idea.xml file from the XWiki code style page.
Did I forgot to configure something in addition to simply import the codestyle file ?

Reply:
This file is very old now and I guess is not matching fully our coding practices. Someone would need to check it and fix it.

Reply:
I looked at the whole codebase and a found many 8 spaces indentations, but 12 spaces indentations as well. It would make sense if the formatting style from wrapped exceptions declaration has changed at some point.

Reply:
We’ve always use 4 spaces for java as our code style. And 2 for xml.

Reply:
Started a thread at https://forum.xwiki.org/t/checkstyle-adjustments/6642